docs/dev_guide/pr_review_procedure.rst

```diff
@@ -0,0 +1,81 @@
+This document describes the procedure that the people in the
+sunpy-maintainters group must follow when merging a pull request. If in
+any doubt about merging a pull request, the correct course of action is
+to contact the Lead Developer / Executive Director and ask.
+
+Fundamental Components of a Pull Request
+========================================
+
+Each pull request *must* meet the following criteria before it is
+considered for merge:
+
+-  The code must be PEP 8 compliant and meet the ill-defined SunPy
+   quality standards.
+-  The PR must contain a CHANGELOG entry if a sunpy-maintainter deems so.
```

Comment:
We should formalise this a little. Only changes that don't affect the behaviour of code?!?